ross/results.py

```diff
@@ -1072,39 +1000,14 @@ def plot_phase(self, inp, out, **phase_kwargs):
         )
 
         fig.update_xaxes(
-            title_text="<b>Frequency</b>",
-            title_font=dict(family="Arial", size=20),
-            tickfont=dict(size=16),
-            range=[0, np.max(frequency_range)],
-            gridcolor="lightgray",
-            showline=True,
-            linewidth=2.5,
-            linecolor="black",
-            mirror=True,
-        )
-        fig.update_yaxes(
-            title_text="<b>Phase</b>",
-            title_font=dict(family="Arial", size=20),
-            tickfont=dict(size=16),
-            gridcolor="lightgray",
-            showline=True,
-            linewidth=2.5,
-            linecolor="black",
-            mirror=True,
-        )
-        fig.update_layout(
-            legend=dict(
-                font=dict(family="sans-serif", size=14),
-                bgcolor="white",
-                bordercolor="black",
-                borderwidth=2,
-            ),
-            **phase_kwargs,
+            title_text="<b>Frequency</b>", range=[0, np.max(frequency_range)]
```

Comment:
I think we have to fix this range. I did a bad default range, setting to begin at 0. If the user does not input a frequency_range starting at 0, the plot x axis will still start at 0.
I fixed it in run_campbell, but I let it pass in other classes.

Change
`range=[0, np.max(frequency_range)]`
to 
`range=[np.min(frequency_range), np.max(frequency_range)]`
or similar

Reply:
I will do this now.